tests/data_catalog/test_data_catalog.py

```diff
@@ -713,16 +694,6 @@ def test_to_stac(self, data_catalog: DataCatalog):
 
         raster_stac_catalog.add_item(raster_stac_item)
 
-        outcome = cast(
-            StacCatalog, source.to_stac_catalog(on_error=ErrorHandleMethod.RAISE)
-        )
-
-        assert raster_stac_catalog.to_dict() == outcome.to_dict()  # type: ignore
-        source.metadata.crs = (
-            -3.14
-        )  # manually create an invalid adapter by deleting the crs
-        assert source.to_stac_catalog(on_error=ErrorHandleMethod.SKIP) is None
```

Comment:
Why remove the assertions?

Reply:
I didn't really read them properly and thought they were mostly making asserions about the coerce option which no longer exists, I'll put them back